src/libs/actions/User.js

```diff
@@ -14,6 +16,22 @@ function getBetas() {
     });
 }
 
+/**
+ * Modifies a given login so that it matches a valid email format.
+ *
+ * @param {String} login
+ *
+ * @return {String}
+ */
+function normalizeLogin(login) {
+    if (Str.isValidEmail(login)) {
+        return login;
+    }
+
+    return login.charAt(0) === '+' ? `${login}@${CONST.SMS_DOMAIN}` : `+${login}@${CONST.SMS_DOMAIN}`;
```

Comment:
NAB: I like using _.startsWith over .charAt(0). Technically _.startsWith is faster, but obviously this comes down to preference. (It almost felt too small to even mention)